axelor-contract/src/main/resources/views/Contract.xml

```diff
@@ -64,7 +67,8 @@
             <editor x-viewer="true" x-show-on-new="true" x-show-titles="true">
               <panel colSpan="6">
                 <panel hideIf="statusSelect >= 3" colSpan="12">
-                  <field name="supposedActivationDate"/>
+                  <field name="supposedActivationDate"
+                    onChange="action-method-contract-version-check-supposed-activation-date,save,action-method-contract-version-update-contract-lines"/>
```

Comment:
please check this, we should only save, then call a single action method, not call a second action method.